src/app/modules/user-manage/user-manage-list/user-manage-list.component.ts

```diff
@@ -0,0 +1,142 @@
+import { SelectionModel } from '@angular/cdk/collections';
+import { Component, ViewChild } from '@angular/core';
+import { MatPaginator } from '@angular/material/paginator';
+import { MatSort } from '@angular/material/sort';
+import { MatTableDataSource } from '@angular/material/table';
+import { IUser, User } from '../../../models/user.model';
+
+const USERS: IUser[] = [
+  new User({
+    id: 0,
+    email: 'john@email.com',
+    name: 'John',
+    blocked: false,
+    date: new Date(),
+  }),
+  new User({
+    id: 1,
+    email: 'steave@email.com',
+    name: 'Steave',
+    blocked: false,
+    date: new Date(),
+  }),
+  new User({
+    id: 2,
+    email: 'mike@email.com',
+    name: 'Mike',
+    blocked: false,
+    date: new Date(),
+  }),
+  new User({
+    id: 3,
+    email: 'george@email.com',
+    name: 'George Washington Junior',
+    blocked: false,
+    date: new Date(),
+  }),
+  new User({
+    id: 4,
+    email: 'ivan@email.com',
+    name: 'Ivan',
+    blocked: true,
+    date: new Date(),
+  }),
+  new User({
+    id: 5,
+    email: 'hrystyna@email.com',
+    name: 'Hrystyna',
+    blocked: false,
+    date: new Date(),
+  }),
+  new User({
+    id: 6,
+    email: 'alyona@email.com',
+    name: 'Alyona',
+    blocked: true,
+    date: new Date(),
+  }),
+];
+
+@Component({
+  selector: 'app-user-manage-list',
+  templateUrl: './user-manage-list.component.html',
+  styleUrls: ['./user-manage-list.component.scss'],
+})
+export class UserManageListComponent {
+  usersList = new MatTableDataSource(USERS);
```

Comment:
`Recommendation`: Add access modifiers for all fields and methods, it makes you're code more readable